src/main/java/com/calculator/calculation/PostfixConverter.java

```diff
@@ -0,0 +1,76 @@
+package com.calculator.calculation;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.EmptyStackException;
+import java.util.Stack;
+public class PostfixConverter {
+    private ArrayList<String> postfixExpression = new ArrayList<>(); // 후위표기식 저장
+    private String operandPattern = "\\d+";
+
+    private boolean isOperand(String value){
+        if(value.matches(operandPattern)){
+            return true;
+        }
+        else{
+            return false;
+        }
+    }
+
+    public void convertToPostfix(String expression){
+        String infixExpression[] = expression.split(" ");
```

Comment:
SonarLint가 아래와 같은 이유로 다음과 같이 바꾸라고 하네요. (플러그인 설치하셨죠?? 설치 안했다면... 당장 설치 ㄱㄱ)

> Array designators should always be located on the type for better code readability. Otherwise, developers must look both at the type and the variable name to know whether or not a variable is an array.

```suggestion
        String[] infixExpression = expression.split(" ");
```